README.en.md

```diff
@@ -202,7 +202,34 @@ Configure WovnServletFilter to determine request path and query from the same HT
 _The sample request header shown above was referenced from the following site:_
 https://coderwall.com/p/jhkw7w/passing-request-uri-into-request-header
 
-### 2.7. ignoreClasses
+### 2.7. ignorePaths
+
+A comma-separated list of URL path for which you would like WOVN to not translation content withing given directories.
+
+Ignored paths and their contents will not be processed by WovnServletFilter, and will not be sent to Wovn.io for translation. 
+
+For instance, if you want to not translation the admin directory of your website, you should configure as below.
+
+```XML
+<init-param>
+  <param-name>ignorePaths</param-name>
+  <param-value>/admin,/wp-admin</param-value>
+</init-param>
+```
+With this configuration, WOVN.java will ignore the following URLs
+```Text
+https://my-wesite.com/admin
+https://my-wesite.com/admin/
+https://my-website.com/admin/plugin.html
```

Comment:
The example also has `/wp-admin`.
So, I can understand easily, if there are examples with `/wp-admin`.